conda_forge_tick/update_recipe/v2/yaml.py

```diff
@@ -0,0 +1,22 @@
+import io
+from pathlib import Path
+
+from ruamel.yaml import YAML
+
+yaml = YAML()
+yaml.preserve_quotes = True
+yaml.width = 4096
```

Comment:
This is pretty big. The bot uses 320 elsewhere - https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/recipe_parser/_parser.py#L46

Reply:
I was under the impression that we would never want word wrapping where it wasn't done by the user, but sure, can use the suggestion :)

Reply:
I tuned this carefully after several complaints to do enough wrapping to make recipes legible, but not enough to annoy folks. :)